salt/search/config/etc-syslog-ng-conf.d-search.conf

```diff
@@ -0,0 +1,43 @@
+@version: 3.5
+
+source s_search_application_debug {
+    file("/srv/search/var/logs/debug.log"
+         follow_freq(1)
+         program_override("search")
+         flags(no-parse)
+    );
+};
+
+source s_search_application_error {
```

Comment:
the second file is a subset of this one, so only the first should be uploaded; otherwise we get the errors appearing twice